sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -267,6 +267,7 @@ object SQLConf {
       "Note: For structured streaming, this configuration cannot be changed between query " +
       "restarts from the same checkpoint location.")
     .intConf
+    .checkValue(_ > 0, "the value of sql.shuffle.partitions must be greater than 0")
```

Comment:
```diff
- .checkValue(_ > 0, "the value of sql.shuffle.partitions must be greater than 0")
+ .checkValue(_ > 0, "The value of spark.sql.shuffle.partitions must be greater than 0")
```

Seems `SHUFFLE_PARTITIONS.key` can't be referred before defining `SHUFFLE_PARTITIONS`.